zst/vector/io.go

```diff
@@ -1,35 +1,30 @@
-// Package column implements the organization of columns on storage for a
-// ZST columnar storage object.
+// Package vector implements the organization of Zed data on storage as
+// vectors in a ZST vector storage object.
 //
 // A ZST object is created by allocating a Writer for any top-level Zed type
 // via NewWriter.  The object to be written is wrapped
-// in a Spiller with a column threshold.  Output is streamed to the underlying spiller
-// in a single pass.  (In the future, we may implement multiple passes to optimize
-// the storage layout of column data or spread a given ZST object across multiple
-// files.
+// in a Spiller with a vector threshold.  Output is streamed to the underlying spiller
+// in a single pass.
 //
 // NewWriter recursively decends into the Zed type, allocating a Writer
 // for each node in the type tree.  The top-level body is written via a call
-// to Write.  The columns buffer data in memory until they reach their
+// to Write.  The vectors buffer data in memory until they reach their
 // byte threshold or until Flush is called.
 //
-// After all of the Zed data is written, a reassembly map is formed for
-// each column writer by calling its EncodeMap method, which builds the
-// value in place using zcode.Builder and returns the Zed type of
-// the reassembly map value.
+// After all of the Zed data is written, a metadata section is written consisting
+// of reassembly segments for each vector, which is called with the Metadata
+// method on the zst.Writer.
```

Comment:
This could stand a little rephrasing. Specifically, the subject of the phrase "which is called..." isn't very clear.